<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicLrcMakerWidget</class>
 <widget class="QWidget" name="MusicLrcMakerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>718</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>歌词制作</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>711</width>
     <height>621</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topTitleName">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>8</y>
     <width>220</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:&quot;#FFFFFF&quot;;font-weight:bold</string>
   </property>
   <property name="text">
    <string>歌词制作</string>
   </property>
  </widget>
  <widget class="QToolButton" name="topTitleCloseButton">
   <property name="geometry">
    <rect>
     <x>691</x>
     <y>11</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>35</y>
     <width>711</width>
     <height>590</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageMain">
    <widget class="QLabel" name="label_M2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>670</width>
       <height>315</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_background1</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_M">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>371</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_title</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_M1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>451</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>只需轻轻按动手指，就能制作出卡拉OK的精准桌面歌词，还能加入你的个性化信息。</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>187</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#0091ff;&quot;&gt;动感歌词制作方法&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>190</y>
       <width>18</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_arrow</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_M5">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>280</y>
       <width>0</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;1.按歌词节奏敲击&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff5500;&quot;&gt;右键头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M6">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>315</y>
       <width>0</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;2.歌词有停顿敲击&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff5500;&quot;&gt;下键头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M7">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>350</y>
       <width>0</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;3.回到上一字敲击&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff5500;&quot;&gt;左键头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M8">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>385</y>
       <width>0</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;4.回到上一行敲击&lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff5500;&quot;&gt;上键头&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_M9">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>420</y>
       <width>0</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;5.暂停/继续敲击 &lt;/span&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ff5500;&quot;&gt;空格键&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton_M">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_M">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_M_Up">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>73</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/lrc/lb_make_up);border-style: inset;</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_M_Up_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>340</y>
       <width>73</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/lrc/lb_make_left);border-style: inset;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_M_Up_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>340</y>
       <width>73</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/lrc/lb_make_down);border-style: inset;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_M_Up_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>239</x>
       <y>340</y>
       <width>73</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/lrc/lb_make_right);border-style: inset;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="button_M_Up_5">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>400</y>
       <width>251</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/lrc/lb_make_space);border-style: inset;</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageFirst">
    <widget class="QLabel" name="label_F">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>650</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_first</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_F1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_background2</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="stateButton_F">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>112</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="MusicMovingClickedSlider" name="timeSlider_F">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>115</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QTextEdit" name="lrcTextEdit">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>435</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="artNameEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="authorNameEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_F5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>262</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>制作者：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="introductionTextEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>150</width>
       <height>230</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_F3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>202</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>歌曲名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>232</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>歌手名：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>292</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>说　明：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_F2">
     <property name="geometry">
      <rect>
       <x>251</x>
       <y>175</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入着正确的歌词文本，否则会制作失败！</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="songNameEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="previousButton_F">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton_F">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_F">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime_F">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>112</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00/00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageSecond">
    <widget class="QLabel" name="label_S">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>650</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_second</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_S1">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>650</width>
       <height>87</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_background3</pixmap>
     </property>
    </widget>
    <widget class="MusicMovingClickedSlider" name="timeSlider_S">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>541</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_S">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previousButton_S">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nextButton_S">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>下一步</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime_S">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>197</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00/00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QTextEdit" name="makeTextEdit">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>240</y>
       <width>660</width>
       <height>280</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageThird">
    <widget class="QLabel" name="label_T">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>40</y>
       <width>650</width>
       <height>36</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_third</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_T1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>460</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;恭喜你，歌词制作完成，记得上传歌词分享哦&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="previousButton_T">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>上一步</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton_T">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>本地保存</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelButton_T">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="MusicVLayoutAnimationWidget" name="lrcViewer" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>650</width>
       <height>250</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#9DBBD4;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remakeButton_T">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>重新制作</string>
     </property>
    </widget>
    <widget class="QPushButton" name="uploadButton_T">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>540</y>
       <width>90</width>
       <height>30</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>上传歌词</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelTime_T">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>170</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>00:00/00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_T2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>158</y>
       <width>651</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_background2</pixmap>
     </property>
    </widget>
    <widget class="MusicMovingClickedSlider" name="timeSlider_T">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>173</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="stateButton_T">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_T3">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>103</y>
       <width>38</width>
       <height>38</height>
      </rect>
     </property>
     <property name="pixmap">
      <pixmap resource="../../../../TTKUi/TTKModule.qrc">:/lrc/lb_make_finish</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_T4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>650</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background:#9DBBD4;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="lrc_make_up_T">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>280</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="lrc_make_down_T">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>28</width>
       <height>28</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
    </widget>
    <zorder>lrcViewer</zorder>
    <zorder>label_T</zorder>
    <zorder>label_T1</zorder>
    <zorder>previousButton_T</zorder>
    <zorder>saveButton_T</zorder>
    <zorder>cancelButton_T</zorder>
    <zorder>remakeButton_T</zorder>
    <zorder>uploadButton_T</zorder>
    <zorder>label_T2</zorder>
    <zorder>timeSlider_T</zorder>
    <zorder>stateButton_T</zorder>
    <zorder>labelTime_T</zorder>
    <zorder>label_T3</zorder>
    <zorder>label_T4</zorder>
    <zorder>lrc_make_down_T</zorder>
    <zorder>lrc_make_up_T</zorder>
   </widget>
  </widget>
  <zorder>background</zorder>
  <zorder>topTitleCloseButton</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>topTitleName</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MusicMovingClickedSlider</class>
   <extends>QSlider</extends>
   <header>musicmovingclickedslider.h</header>
  </customwidget>
  <customwidget>
   <class>MusicVLayoutAnimationWidget</class>
   <extends>QWidget</extends>
   <header location="global">musiclayoutanimationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../TTKUi/TTKModule.qrc"/>
 </resources>
 <connections/>
</ui>
